### PR TITLE
Use symbolic constants for HTTP codes

### PR DESCRIPTION
**Pull request by nathanielmanistaatgoogle**
I figured why not do something really easy for my first substantive change to this codebase. :-P

### CODE REVIEW

**Comment by nathanielmanistaatgoogle** [comment]
(Oh, and _of course_ this may wait until after the more important work we're doing.)

File: oauth2client/client.py [hunk 1]
```diff
@@ -73,7 +74,7 @@
 OOB_CALLBACK_URN = 'urn:ietf:wg:oauth:2.0:oob'
 
 # Google Data client libraries may need to set this to [401, 403].
-REFRESH_STATUS_CODES = [401]
+REFRESH_STATUS_CODES = [http_client.UNAUTHORIZED]
```
Comment (dhermes) [inline oauth2client/client.py]: Use a tuple instead of a list. (You break it you buy it)

**Comment by dhermes** [comment]
LGTM pending Travis / small change from my comment

File: oauth2client/client.py (hunk 1 above)
Comment (nathanielmanistaatgoogle) [inline oauth2client/client.py]: Done and repushed.

**Pull request merged**